lib/include/hse_ikvdb/kvs_rparams.h

```diff
@@ -70,8 +70,10 @@ struct kvs_rparams {
     uint64_t capped_evict_ttl;
 
     struct {
-        enum vcomp_default deflt;
-    } compression;
+        struct {
+            enum vcomp_default deflt;
```

Comment:
nit: I'm used to seeing "dflt" being used as an abbreviation for default.. "deflt" seemed a bit odd...

Reply:
I will make that change